src/views/Portal/components/ApplicationForm/index.view.tsx

```diff
@@ -34,6 +37,16 @@ const ApplicationForm: React.FC = () => {
     setConnectedFormData,
   } = useApplication()!
 
+  const { getAccessTokenSilently } = useAuth0()
+  const [token, setToken] = useState("no token")
+  const [successOnSubmit, setSubmitStatus] = useState("none")
+
+  useEffect(() => {
+    getAccessTokenSilently()
+      .then(ac => setToken(ac))
+      .catch()
+  }, [])
```

Comment:
You can grab the access token from useApplication() so we don't have to rerequest.